core/templates/domain/exploration/SolutionObjectFactorySpec.ts

```diff
@@ -129,12 +129,25 @@ describe('Solution object factory', () => {
         'One solution is "1". This is the explanation to the answer.');
 
       solution.setCorrectAnswer([
-        ['<p>1</p>', '<b>3</b>'],
-        ['c', '<oppia-noninteractive-math></oppia-noninteractive-math>']
+        ['Choice 1'], ['Choice 2']
```

Comment:
It might make sense to add options with HTML tags, and perhaps math latex string.

Reply:
added p tag